### ISSUE TITLE
F-Droid Metadata

### ISSUE THREAD

**Issue by Nizami20052022** [TeamNewPipe/NewPipe#9288]
### Checklist

- [X] I made sure that there are *no existing issues* - [open](https://github.com/TeamNewPipe/NewPipe/issues) or [closed](https://github.com/TeamNewPipe/NewPipe/issues?q=is%3Aissue+is%3Aclosed) - which I could contribute my information to.
- [X] I have read the [FAQ](https://newpipe.net/FAQ/) and my problem isn't listed.
- [X] I'm aware that this is a request for NewPipe itself and that requests for adding a new service need to be made at [NewPipeExtractor](https://github.com/TeamNewPipe/NewPipeExtractor/issues).
- [X] I have taken the time to fill in all the required details. I understand that the feature request will be dismissed otherwise.
- [X] This issue contains only one feature request.
- [X] I have read and understood the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/dev/.github/CONTRIBUTING.md).

### Feature description

Add to F-Droid Metadata in Azerbaijani language. I translated it on Weblate

### Why do you want this feature?

The changes can be read in the Azerbaijani language.

**Comment by opusforlife2** [comment]
Already added.

**Comment by Nizami20052022** [comment]
I don't see this language in F-Droid.

**Comment by opusforlife2** [comment]
Indeed: https://gitlab.com/fdroid/fdroiddata/-/tree/master/metadata/org.schabi.newpipe

**Comment by TobiGr** [comment]
Well, translations are sync approximately once or twice a month by f-droid (see https://gitlab.com/fdroid/fdroiddata/-/commits/master?search=sync+translations). But for some reason, our translations have not been synced for four month now.

**Comment by opusforlife2** [comment]
Gonna ping them?

**Comment by Nizami20052022** [comment]
How should I do it?

**Comment by opusforlife2** [comment]
Not you. Tobi. Don't worry, we'll handle it.
